index.js

```diff
@@ -380,24 +387,26 @@ function handleRequests() {
   /** everything was applied correctly; reset the global variable. */
   responseMiddlewareHasBeenApplied = false;
 
-  /** middleware to handle REQUESTS */
-  app.use((req, res, next) => {
-    try {
-      const methodAndPathKey = getMethod(req);
-      if (methodAndPathKey && methodAndPathKey.method && methodAndPathKey.pathKey) {
-        const method = methodAndPathKey.method;
-        updateSchemesAndHost(req);
-        processors.processPath(req, method, methodAndPathKey.pathKey);
-        processors.processHeaders(req, method, spec);
-        processors.processBody(req, method);
-        processors.processQuery(req, method);
+  if (!ignoredNodeEnvironments.includes(process.env.NODE_ENV)) {
```

Comment:
same as above with `if`

Reply:
Same as https://github.com/mpashkovskiy/express-oas-generator/pull/69#discussion_r487410905

Reply:
and here

Reply:
Done